CONTRIBUTING.md

```diff
@@ -107,7 +107,9 @@ cp apps/api/.env.example apps/api/.env
 
 4. Start developing:
 
+
 ```bash
+pnpm build (just once)
```

Comment:
Can you remove `(just once)` hint here (or add it to the comment)? The command won't run.